cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -213,6 +213,15 @@
     </description>
   </property>
 
+  <property>
+    <name>location.cache.force.cleanup.ms</name>
+    <value>3600000</value>
```

Comment:
I think `3600000` is to frequent since it also blocks read during cleanup, I would suggest increasing it to 1 day.